src/lmnt/api.py

```diff
@@ -77,13 +81,142 @@ async def close(self):
       await self._session.close()
     self._session = None
 
-  async def list_voices(self):
+
+  async def list_voices(self, starred: bool = False, owner: str = 'all'):
+    """
+    Returns a list of voices available to you.
+
+    Optional parameters:
+    - `starred`: Show starred voices only. Defaults to `false`.
+    - `owner`: Comma-separated list of voice owners to specify which voices to return. Choose from `lmnt`, `me`, or `all`. Defaults to `all`.
+    """
+    if owner not in ['all', 'me']:
```

Comment:
This disallows `lmnt` but the docs say it's a valid owner option.

Reply:
great catch